modules/apps/depot/depot-api/src/main/java/com/liferay/depot/constants/DepotRolesConstants.java

```diff
@@ -23,4 +23,6 @@ public class DepotRolesConstants {
 
 	public static final String ASSET_LIBRARY_OWNER = "Asset Library Owner";
 
+	public static final String CMS_CONSUMER = "CMS Consumer";
```

Comment:
@AliciaGarciaGarcia, I'm unsure if we will be able to merge it with this name. Most likely we are going to be asked to change to ASSET_LIBRARY_CONSUMER. Still, we can try.....

Reply:
but it's not the asset_library /space consumer

As far I know, it's the CMS consumer, this will have different meaning.

I will ask the team about it.

Reply:
see https://liferay.slack.com/archives/C08DQSVC3U4/p1743669297676289

Reply:
Talked to BChan and we are good to go